### ISSUE TITLE
[Accessibility] -Interactive Vipps elements lacks descriptive alt-text for voice over

### ISSUE THREAD

**Issue by joaooliveiramanteigas** [Adyen/adyen-web#1623]
Describe the bug:

User story
As a customer using screen voice over I am not able to understand the meaning of these fields and buttons

Description
To meet the requirements of WCAG 2.1 ([4.1.2 name, role, value ](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#name-role-value)) links must be described properly for voice over and screen readers.

Suggested solution

Im not sure if this is something controlled by your team or if this resides with Vipps. Let me know if this is something you can handle or if we should handle this directly with Vipps team.

Buttons in vipps payment need a better description.

Cancel `a` tag in vipps payment need to have a better description for voice over.

From:
alt="Cancel"

To:
NO: alt="Avbryt vipps betaling"
EN: alt="Cancel vipps payment"

Next button in vipps payment need to have a better description for voice over.

From:
alt="Next"

To:
NO: alt="Fortsett vipps betaling"
EN: alt="Proceed vipps payment"

Image displaying where the accessibility issues presides.
![image](https://user-images.githubusercontent.com/85570868/172647426-93263f2c-02ff-4ce3-a9dd-36d41ff067f7.png)
![image](https://user-images.githubusercontent.com/85570868/172647907-a0fcb815-71f2-46fd-a036-2c595ef32f32.png)

Smartphone (please complete the following information):

Device: iPhone 8
OS: iOS14.5
Browser: Chrome
Additional Context
We are using the Web Components/Drop-in v5.16.0

Thank you in advance.
